test/read-only-handler.spec.ts

```diff
@@ -304,7 +304,28 @@ describe('ReadOnlyHandler', () => {
             expect( () => {
                 value.foo = '';
             }).toThrow();
```

Comment:
The `toThrow` should validate the error message to ensure it's the expected one.